README.md

```diff
@@ -1,5 +1,3 @@
-# learning-git
-
 **Preconditions:** 
```

Comment:
You shouldn't have any changes to this file, so you should revert your changes here.